### PR TITLE
Dependency Performance Improvements

### PR DESCRIPTION
**Pull request by willfarrell**
Closes: https://github.com/middyjs/middy/issues/436

### CODE REVIEW

**Comment by vladgolubev** [comment]
What about importing only relevant clients from aws-sdk? 

See https://theburningmonk.com/2019/03/just-how-expensive-is-the-full-aws-sdk/

**Comment by willfarrell** [comment]
@vladgolubev Great article, and suggestion.

I'm curious, there is a common hack to enable keep-alive (below, sometimes seen as a layer). Does adding this in lobotomize cold start performance for faster aws service re-connections. If we only include the relevant clients only, does it undo keepAlive? @theburningmonk maybe you've already tested this?

... So many dimensions to test. A project for the new year.

```js
const AWS = require('aws-sdk')
const https = require('https')

const tlsAgent = new https.Agent({
  keepAlive: true,
  maxSockets: 50,
  rejectUnauthorized: true
})
tlsAgent.setMaxListeners(0)

AWS.config.update({
  httpOptions: {
    agent: tlsAgent
  }
})

module.exports = AWS
```

**Comment by vladgolubev** [comment]
@willfarrell see https://theburningmonk.com/2019/02/lambda-optimization-tip-enable-http-keep-alive/

> UPDATE 22/09/2019: big thanks to Joe Bowbeer for mentioning this in the comments. Since [AWS SDK v2.463.0](https://github.com/aws/aws-sdk-js/blob/master/CHANGELOG.md#24630) you no longer need these couple of lines of code change any more. Instead, set the environment variable AWS_NODEJS_CONNECTION_REUSE_ENABLED to 1 to make the SDK reuse connections by default.

File: packages/db-manager/__tests__/index.js [hunk 1]
```diff
@@ -3,7 +3,7 @@ jest.mock('knex')
 
 const knex = require('knex')
 
-const middy = require('@middy/core')
+const middy = require('../../core')
```
Comment (lmammino) [inline packages/db-manager/__tests__/index.js]: any reason for changing this?

Comment (willfarrell) [reply packages/db-manager/__tests__/index.js]: Just to make it consistent with all the other modules. This ensures they're all being tested with the same version of core as the rest.

**Pull request merged**